tasks_downstream.py

```diff
@@ -108,7 +108,33 @@ def write_code_workspace_file(c, cw_path=None):
         pass  # Nevermind, we start with a new config
     # Static settings
     cw_config.setdefault("settings", {})
-    cw_config["settings"].update({"search.followSymlinks": False})
+    cw_config["settings"].update(
+        {
+            "editor.formatOnSave": True,
+            "python.autoComplete.extraPaths": [f"{str(SRC_PATH)}/odoo"],
+            "python.formatting.blackArgs": [],
+            "python.formatting.provider": "black",
+            "python.languageServer": "Jedi",
+            "python.linting.flake8Enabled": True,
+            "python.linting.ignorePatterns": [f"{str(SRC_PATH)}/odoo/**/*.py"],
```

Comment:
Just to clarify: I tried to use the `${workspaceFolder}` variable here but, for some reason, it only works with this full path :man_shrugging: